apps/search-poc/src/utils/morph.ts

```diff
@@ -0,0 +1,21 @@
+
+import swcParser from 'swcmorphologyparser';
+
+
+const CONVERTER_URL = 'http://morph-service.ocp.bbp.epfl.ch/converter/api';
```

Comment:
can you add this URL to https://github.com/BlueBrain/nexus-js/blob/search-poc/apps/search-poc/src/config.ts#L7

Reply:
Done